requirements.txt

```
@@ -2,7 +2,7 @@ tqdm>=4.66.2
 openai>=1.23.6
 tiktoken>=0.6.0
 transformers>=4.40.1
-pyserini>=0.35.0
+pyserini==0.39.0
```

```suggestion
pyserini==0.42.0
```

This is an important push and removes extra dependencies, can you remove faiss-cpu and see if it works?

---

Pyserini 0.42.0 refactored things that break rank_llm. Currently pushed an ugly solution for rank_llm to work, but would be better to update on pyserini's side as well